data-pipeline/tests/unit/pre_processing/test_aar.py

```diff
@@ -6,114 +6,98 @@
 
 def test_aar_data_has_correct_output_columns(prepared_aar_data: pd.DataFrame):
     assert list(prepared_aar_data.columns) == [
+        "LA",
+        "Estab",
         "Academy UPIN",
-        "In year balance",
-        "PFI School",
-        "Trust UPIN",
-        "Income_Other DFE grants",
-        "Income_SEN funding",
+        "Date joined or opened if in period",
+        "Date left or closed if in period",
+        "BNCH11110T (EFA Revenue Grants)",
+        "BNCH11131 (DfE Family Revenue Grants)",
+        "Income_Targeted grants",
         "Income_Pre Post 16",
         "Income_Other grants",
         "Income_Government source",
         "Income_Academies",
         "Income_Non government",
-        "Income_Facilities and services",
+        "BNCH11123-BAI011-A (Academies - Income)",
```

Comment:
Should we be passing the raw data references to our outputs? Shouldn't they be mapped to a particular cost category?